applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -300,6 +305,14 @@ def __main():
         default=False,
         help="Set when using nRF21540 for extra TX power",
         )
+    parser.add_argument(
+        "-u"
+        "--user_bt_name",
+        action="store_true",
+        dest="user_specific_name",
+        default=False,
+        help="Set to true for user specific BT broadcast name",
```

Comment:
This is not the broadcast name though, this is the advertised name, or 'friendly name'. Since it is hard coded to be NRF5340_AUDIO_[username or something similar], that should be stated. There could be a situation where another company takes our sample and creates a prj.conf specific for them setting CONFIG_BT_DEVICE_NAME=[project_product]. Then they figure out they want their developers to have different BT_DEVICE_NAMEs and uses this, suddenly all their kits will be 'NRF5340_AUDIO_....'